tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/CoreJSLibs.scala

```diff
@@ -111,7 +111,8 @@ private[scalajs] object CoreJSLibs {
       else "" // blank line preserves line numbers in source maps
     }
 
-    val content = lines.mkString("", "\n", "\n")
+    val content = lines.mkString("", "\n", "\n").
+        replace("{{LINKER_VERSION}}", ScalaJSVersions.current)
```

Comment:
`replace(` should go at the end of the previous line.
Or the `.` should go at the beginning of the second line, with 2 spaces.